emr-logservice/src/main/java/org/apache/spark/streaming/aliyun/logservice/utils/VersionInfoUtils.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.spark.streaming.aliyun.logservice.utils;
+
+import java.io.InputStream;
+import java.util.Properties;
+
+public class VersionInfoUtils {
+    private static final String VERSION_INFO_FILE = "spark-versioninfo.properties";
```

Comment:
It is better to use "versioninfo.properties" instead of "spark-versioninfo.properties"

Reply:
other sdk have this file: versioninfo.properties

Reply:
or you can use "emr-logservice-versioninfo.properties" instead.

Reply:
ok